ui/raidboss/data/06-ew/raid/p1s.txt

```diff
@@ -0,0 +1,105 @@
+### P1S: Asphodelos: The First Circle (Savage)
+#
+# -it 'Erichthonios' -oc 'Erichthonios' -p "6629:14.1"
```

Comment:
Are there any `-ii` ignored abilities?

Reply:
There could have been a couple added but I just manually cleaned up the timeline instead since it needed a ton of cleanup anyways to remove duplicates.

Reply:
Here are some I believe will be wanted:
# -ii 65F6 65FA 65FD 65FF 6602 6607 660A 660B 660D 6610 6611 6624